### PR TITLE
More sql queries to:

### PR DESCRIPTION
**Pull request by cthanos**
Create table users & drop columns 'username' and 'password' from table person

### CODE REVIEW

File: DB/academicmanager.sql [hunk 1]
```diff
@@ -101,8 +102,8 @@ ALTER TABLE `course`
 --
 -- Ευρετήρια για πίνακα `person`
 --
-ALTER TABLE `person`
-  ADD PRIMARY KEY (`personId`);
+/*ALTER TABLE `person`
```
Comment (gmetal) [inline DB/academicmanager.sql]: Remove completely - not needed any more

File: DB/academicmanager.sql [hunk 2]
```diff
@@ -46,7 +46,8 @@ CREATE TABLE `person` (
   `taxNumber` varchar(128) COLLATE utf8_bin NOT NULL,
   `bankAccount` varchar(128) COLLATE utf8_bin NOT NULL,
   `username` varchar(128) COLLATE utf8_bin NOT NULL,
-  `password` varchar(128) COLLATE utf8_bin NOT NULL
+  `password` varchar(128) COLLATE utf8_bin NOT NULL,
```
Comment (gmetal) [inline DB/academicmanager.sql]: Since we are dropping the column later on, no need to add it

File: DB/academicmanager.sql [hunk 3]
```diff
@@ -124,3 +125,23 @@ ALTER TABLE `course`
 --
 ALTER TABLE `person`
   MODIFY `personId` int(11) NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=13;
+
+
+-- DROP the columns below as requested by G.Metaxas
+ALTER TABLE person DROP username,DROP password;
```
Comment (gmetal) [inline DB/academicmanager.sql]: Split this into two separate statements and add an if exists condition: eg ALTER TABLE Person DROP COLUMN IF EXISTS USERNAME:

File: DB/academicmanager.sql [hunk 4]
```diff
@@ -44,35 +44,34 @@ CREATE TABLE `person` (
...
-(7, 'Bruce', 'Lee', 'Teacher', '1951-01-03', 'Male', 'brulee@gmail.com', '4456789123', 'Patision 66', '6789005432', 'GB55879000001155662', 'brulee', '12345678'),
-(8, 'Mike', 'Tyson', 'Teacher', '1965-01-15', 'Male', 'miktys@gmail.com', '0090807050', 'Heaven 32', '9087654321', 'US77890110000530099112', 'miktys', '12345678'),
-(9, 'Madonna', 'Madonna', 'Student', '1967-11-13', 'Female', 'madonna@gmail.com', '6968674321', 'Sunset 88', '2324257610', '', 'madonna', '12345678'),
-(10, 'Tony', 'Montana', 'Teacher', '2048-02-13', 'Male', 'drugs@gmail.com', '6909001234', 'Cartel 9', '2134567890', 'GR0022375812093467', 'cocaine', '12345678'),
-(11, 'Luke', 'Skywalker', 'Admin', '1955-01-28', 'Male', 'jedi@gmail.com', '6932425261', 'Rebel Camp 4', '2341538753', 'GR238719094328734', 'admin', 'admin'),
-(12, 'Darth', 'Vader', 'Student', '1945-06-18', 'Male', 'sithlord@yahoo.com', '6974358721', 'Deathstar 10', '1452348790', '', 'sith', '12345678');
+INSERT INTO `person` (`personId`, `name`, `surname`, `type`, `dateOfBirth`, `sex`, `email`, `phoneNumber`, `address`, `taxNumber`, `bankAccount`) VALUES
```
Comment (gmetal) [inline DB/academicmanager.sql]: You need to execute DELETE FROM person; before the insert, or you will get a duplicate PK error

File: DB/academicmanager.sql [hunk 5]
```diff
@@ -44,35 +44,34 @@ CREATE TABLE `person` (
   `phoneNumber` varchar(128) COLLATE utf8_bin NOT NULL,
   `address` varchar(128) COLLATE utf8_bin NOT NULL,
   `taxNumber` varchar(128) COLLATE utf8_bin NOT NULL,
-  `bankAccount` varchar(128) COLLATE utf8_bin NOT NULL,
-  `username` varchar(128) COLLATE utf8_bin NOT NULL,
-  `password` varchar(128) COLLATE utf8_bin NOT NULL
+  `bankAccount` varchar(128) COLLATE utf8_bin NOT NULL
```
Comment (gmetal) [inline DB/academicmanager.sql]: Add ```,``` at the end of the line

File: DB/academicmanager.sql [hunk 6]
```diff
@@ -44,35 +44,34 @@ CREATE TABLE `person` (
...
+(12, 'Darth', 'Vader', 'Student', '1945-06-18', 'Male', 'sithlord@yahoo.com', '6974358721', 'Deathstar 10', '1452348790', '');
 
 -- --------------------------------------------------------
 
 --
 -- Δομή πίνακα για τον πίνακα `teaching`
 --
 
-CREATE TABLE `teaching` (
+CREATE TABLE IF NOT EXISTS `teaching` (
```
Comment (gmetal) [inline DB/academicmanager.sql:83]: Add primary key here

File: DB/academicmanager.sql [hunk 7]
```diff
@@ -99,12 +98,6 @@ ALTER TABLE `course`
...
 --
--- Ευρετήρια για πίνακα `person`
---
-ALTER TABLE `person`
-  ADD PRIMARY KEY (`personId`);
-
---
 -- Ευρετήρια για πίνακα `teaching`
 --
 ALTER TABLE `teaching`
```
Comment (gmetal) [inline DB/academicmanager.sql]: remove

File: DB/academicmanager.sql [hunk 8]
```diff
@@ -124,3 +117,24 @@ ALTER TABLE `course`
...
+
+CREATE TABLE IF NOT EXISTS users (
+id int NOT NULL,
+user_email varchar (255) NOT NULL,
+user_password char (128),
+last_login TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ,
+date_created TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+status_account boolean,
+personId int (11),
+PRIMARY KEY(user_email),
```
Comment (gmetal) [inline DB/academicmanager.sql]: The primary key should be id, not user_email. User_email may be an indexed column/unique constraint, etc

Comment (cthanos) [reply DB/academicmanager.sql]: OK

File: DB/academicmanager.sql (hunk 7 above)
Comment (cthanos) [reply DB/academicmanager.sql]: OK

File: DB/academicmanager.sql (hunk 6 above)
Comment (cthanos) [reply DB/academicmanager.sql:83]: OK

File: DB/academicmanager.sql (hunk 4 above)
Comment (cthanos) [reply DB/academicmanager.sql]: OK

File: DB/academicmanager.sql (hunk 5 above)
Comment (cthanos) [reply DB/academicmanager.sql]: OK

**Review by gmetal — APPROVED** [review]
LGTM

**Pull request merged**